scripts/staging.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# For now, we use this script to start our staging preview on render
+# by changing the ENTRYPOINT and CMD at the dockerfile to this.
+
+# example usages:
+# ./staging.sh
+# ./staging.sh https://example.com
+# ./staging.sh https://example.com:8080
+
+# If no parameter is passed, use https://staging.bytebase.com as host and 80 as port by default
+ONLINE_DEMO_HOST='https://staging.bytebase.com'
+ONLINE_DEMO_PORT='80'
+if [ $1 ]; then
+    PROTOCAL=`echo $1 | awk -F ':' '{ print $1 }'`
+    URI=`echo $1 | awk -F '[/:]' '{ print $4; }'`
+    PORT=`echo $1 | awk -F '[/:]' '{ print $5; }'`
+
+    ONLINE_DEMO_HOST=$PROTOCAL://$URI
+
+    if [ $PORT ]; then
+        ONLINE_DEMO_PORT=$PORT
+    fi
+fi
+
+echo "Starting Bytebase in demo mode at ${ONLINE_DEMO_HOST}:${ONLINE_DEMO_PORT}..."
```

Comment:
Since we don't need to seed the data and restart, do we actually need to have this script?

Reply:
Since we build a release version, it would not seed any data by default.

Reply:
PTAL